cves/CVE-2010-1157.yml

```diff
@@ -149,15 +140,11 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this
     section by explaining what happened between the VCCs and the fix.
-  answer: |
-    The git history for this directory only goes back to a branching of Tomcat 5.5.x. Between
-    that commit and the fix for the issue, there is a commit that mentions a fix for session fixation.
-    Apparently session fixation is an attack that alllows a hacker to hijack a user session. The commit
-    aimed to prevent this by providing the option to change session ID upon authentication.
+  answer: Between the VCC's and the fix, there did not appear to be any interesting commits.
```

Comment:
I understand there was nothing of interest, but this does not show you completed this step. Provide an explanation of what happened between the VCC and the fix as stated.

Reply:
Provided an explanation.